packages/terra-section-header/docs/README.md

```diff
@@ -0,0 +1,34 @@
+# Terra Section Header
+
+Section Header presentational component that provides an onClick callback and animateable status icon.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-section-header`
+
+## Usage
+
+```jsx
+import React from 'react';
+import SectionHeader from 'terra-section-header';
+
+<SectionHeader
+  title="Section Header Example"
+  isAnimated
+  onClick={() => {}}
+  isOpen={true}
+/>
+```
+
+## Component Features
+
+ * [Cross-Browser Support](https://github.com/cerner/terra-core/wiki/Component-Features#cross-browser-support)
+ * [LTR/RTL Support](https://github.com/cerner/terra-core/wiki/Component-Features#ltr--rtl-support)
+ 
+<!-- Uncomment supported features.
+ * [Responsive Support](https://github.com/cerner/terra-core/wiki/Component-Features#responsive-support)
+ * [Mobile Support](https://github.com/cerner/terra-core/wiki/Component-Features#mobile-support)
+ * [Internationalization Support](https://github.com/cerner/terra-core/wiki/Component-Features#internationalization-i18n-support)
+ * [Localization Support](https://github.com/cerner/terra-core/wiki/Component-Features#localization-support)
+ -->
```

Comment:
Can you remove this commented-out section?

Reply:
I chose to leave it, as neil currently has future plans for this component that might end up adding some of these things, I don't know. It's also what we do with [`terra-form-input`](0cff014cb138b5ea28c281a727e3eb1e202c5fc4).

> Uncomment

😉